### PR TITLE
RavenDB-14585 Do not allow to modify Migrator Path if defined in sett…

### PR DESCRIPTION
**Pull request by Danielle9897**
…ings.json

### CODE REVIEW

File: src/Raven.Studio/typescript/commands/database/tasks/getMigratorPathConfigurationCommand.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import commandBase = require("commands/commandBase");
+import endpoints = require("endpoints");
+
+class getMigratorPathConfigurationCommand extends commandBase {
+
+    execute(): JQueryPromise<Configuration> {
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/tasks/getMigratorPathConfigurationCommand.ts]: from where this Confgiuration type comes from?

File: src/Raven.Studio/typings/_studio/dto.d.ts [hunk 2]
```diff
@@ -643,3 +643,7 @@ interface rawStackTraceResponseItem {
 }
 
 type indexStatus = "Normal" | "ErrorOrFaulty" | "Stale" | "Paused" | "Disabled" | "Idle";
+
+interface Configuration {
```
Comment (ml054) [inline src/Raven.Studio/typings/_studio/dto.d.ts]: Can we rename this something less generic, like MigratorPathConfiguration

Comment (Danielle9897) [reply src/Raven.Studio/typings/_studio/dto.d.ts]: done (my original thought was that maybe 'configuration' interface will represent other settings in the future)

Comment (ml054) [reply src/Raven.Studio/typings/_studio/dto.d.ts]: but this is migrator specific

**Review by ml054 — APPROVED** [review]

**Pull request merged**